tests/test_humidity_sensor.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+import unittest
+
+import mock
+
+from greenpithumb import humidity_sensor
+from greenpithumb import dht11_exceptions
+
+class HumiditySensorTest(unittest.TestCase):
+
+	def setUp(self):
+		self.mock_dht11 = mock.Mock()
+		self.humidity_sensor = humidity_sensor.HumiditySensor(
+			self.mock_dht11)
+
+	def test_humidity_50(self):
+		"""Value given should be value returned."""
+
+		self.mock_dht11.read().error_code = 0
+		self.mock_dht11.read().humidity = 50.0
```

Comment:
I'm having trouble getting this to work. It appears that with the way you have it set up above the error code and humidity aren't getting assigned values.

I'm also a little confused about what you mean when you say "What I'd expect to happen is that each read() creates a new, independent object that gets thrown away at the end of the line" and why this is an issue.